src/WasmDis.ts

```diff
@@ -448,6 +454,17 @@ export class WasmDisassembler {
     this._lines.push(this._buffer);
     this._buffer = '';
   }
+  private logStartOfBodyFunctionOffset() {
+    if (this.addOffsets) {
+      this._currentBodyFunctionOffset = [this._currentPosition];
```

Comment:
I'd prefer to see an object created for this rather than a two element array. Start and End, or maybe even HeaderStart, FirstInstruction, and End, instead of [0] and [1].

Reply:
I am not an expert here but doesn't it increase the amount of information transfer between web worker and main thread? I am more than willing to do it but is that any concern.

Reply:
Hmm, maybe? I don't think there is much difference in memory size on the heap, but the transfer size might be different if the objects get turned into JSON. If so each object would be around 60 bytes per function, which if I look at a large binary I've tested with could translate into megabytes, but still 10 times smaller than the original binary and far smaller than the disassembly, as well as smaller than the line->byte offset array.

I think using an object here provides a cleaner API, and the fact that we will pass this between workers is not really wasdk's problem and probably not particularly important for performance.

Reply:
Okay got you :) I will change it to object {Start, End} where Start is first instruction. So [Start, End) is breakable